thirdparty/swiftmailer/Swift/Transport/MailTransport.php

```diff
@@ -50,6 +52,9 @@ class Swift_Transport_MailTransport implements Swift_Transport
      */
     public function __construct(Swift_Transport_MailInvoker $invoker, Swift_Events_EventDispatcher $eventDispatcher)
     {
+        Deprecation::withNoReplacement(function () {
+            Deprecation::notice('4.12.0', 'Will be replaced with symfony/mailer', Deprecation::SCOPE_CLASS);
+        });
```

Comment:
I figure this is the appropriate message rather than "Will be removed without equivalent functionality" because symfony/mailer comes with a suported sendmail DSN config out of the box: `sendmail://default`

Reply:
Oh this is for MailTransport ... yeah no replacement cos you can't use `mail()` in symfony/mailer

Message should be along the lines of "There is no mail() in CMS 5, use sendmail or SMTP instead"

Reply:
Just realised this is for MailTransport - yeah no replacement here, there is no `mail()` support in symfony/mailer

Message should be along the lines of "There is no mail() in CMS 5, use sendmail or SMTP instead"

Reply:
Should probably put the same deprecation notices in the MailInvoker + SimpleMailInvoker __construct() as well

Reply:
Ooohhh, I thought this was for sendmail, didn't realise it was for `mail()`.
Updated messages. I've only used the "use sendmail or SMTP instead" for the `Swift_MailTransport` class and just noted there's no replacement for the other two.